content/docs/overview.md

```diff
@@ -204,6 +204,22 @@ my_variable: int // cannot be accessed outside this file.
 ```
 `@(private)` is equivalent to `@(private="package")`.
 
+### Authoring a package
+
+A package is a directory of Odin code files which are marked with the same package header: `package main`
```

Comment:
And here I'd repeat the same phrasing:

A package is a directory of Odin code files, all of which have the same package declaration at the top, e.g. `package main`. Each `.odin` file _must_ have the same package name. A directory cannot contain more than 1 package.

Reply:
As above.